frontend/src/components/GeneInfoSideBar/style.ts

```diff
@@ -15,49 +16,17 @@ export const GeneSummary = styled.div`
   color: black;
 `;
 
-export const GeneSynonymsWrapper = styled.div`
-  display: flex;
-  padding-bottom: 8px;
-  align-items: center;
-`;
-
 export const Label = styled.div`
   ${fontBodyXs}
-  ${(props) => {
-    const colors = getColors(props);
-    const fontWeights = getFontWeights(props);
-
-    return `
-      color: ${colors?.gray[500]};
-      font-weight: ${fontWeights?.regular};
-    `;
-  }}
-`;
-
-export const GeneSynonyms = styled.div`
-  padding: 4px 8px;
-  color: black;
-
-  ${fontBodyXs}
-  ${(props) => {
-    const fontWeights = getFontWeights(props);
 
-    return `
-      font-weight: ${fontWeights?.regular};
-    `;
-  }}
```

Comment:
extracted to `<Synonyms />` component

Reply:
One other thing, we should semantic variables for text colors. Instead of `colors?.gray[500]`, we should use the `palette.text.secondary` value in the theme file. 

We might need to update the `Colors` interface and `getColors` function. The text values are in the MUI theme file, but they may not be exposed in SDS.

Reply:
Awesome feedback! Will implement now 🙏 Thanks, Harley!

And yes! We already have a utility function to get `textPrimary` from MUI theme, so I'll create one for `textSecondary` 👌

Reply:
new diff PTAL 🙏 
https://github.com/chanzuckerberg/single-cell-data-portal/pull/5328/commits/4366fd9d2d77efe4c4cedf1610f0cd13fca43134